src/internal/observable/throwError.ts

```diff
@@ -66,11 +183,17 @@ import { Subscriber } from '../Subscriber';
  * @name throwError
  * @owner Observable
  */
```

Comment:
nitpick: Do we really need to have a TSDoc comment on the implementation? It's not directly callable; only the overload signatures are callable. If there's anything in here that's not in the non-deprecated overload signature, can it be moved there?